documents/tutorials/contiv-policy.html

```diff
@@ -3,14 +3,14 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="Container Networking Tutorial">
+    <meta name="description" content="Advanced Contiv Tutorial">
```

Comment:
There are some more things we can add here. for eg : L3 mode, Service Load Balancer. So i wanted to avoid naming `policy` in the name.